cmd/ddev/cmd/config.go

```diff
@@ -35,6 +52,12 @@ var ConfigCommand = &cobra.Command{
 		err = c.Write()
 		if err != nil {
 			util.Failed("Could not write ddev config file: %v\n", err)
+
+		}
+
+		// If a provider is specified, prompt about whether to do an import after config.
+		if provider != "" {
+			util.Success("Configuration complete. You may now run `ddev start` or `ddev pull`")
```

Comment:
Scratch that, I am actually not making this change in this PR.

I find it clear as is, but I'm still happy to change it if it's something you and/or others are passionate about, but a quick grep for "`ddev" shows 80 occurrences of this pattern, including many places not touched by this work. If we make this change, we should make it everywhere for consistency, and doing that across so many files is probably best to do in a follow-up with clear intent, rather than just adding noise to the work here.